### ISSUE TITLE
Dart Sass 2.0.0 support: "Using / for division is deprecated and will be removed in Dart Sass 2.0.0."

### ISSUE THREAD

**Issue by ktan88** [csshugs/FluidMS#41]
Hey there,

We have moved over to Dart Sass, away from Node Sass.
Dart Sass is throwing some deprecation warnings for the / division operator:

```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
Recommendation: math.div($px, $base-font-size)
More info and automated migrator: https://sass-lang.com/d/slash-div
```
See breaking changes:  
https://sass-lang.com/documentation/breaking-changes/slash-div

**Comment by fuhlig** [comment]
#42

**Comment by csshugs** [comment]
Hey @ktan88 ,
I know, it’s been a long time, but I just released version 1.0.0, which provides a Dart Sass migration.

Let me know when somethings not working as expected.
